File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/NodesBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 * @date 2023/2/12
 */
@NoArgsConstructor
@Data
public class NodesBean {
    private String shape;
    private String title;
    private Boolean moduleName;
    private Integer x;
    private Integer y;
    private String id;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/NodesBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */


@AllOpen
class NodesBean {
    var shape: String? = null
    var title: String? = null
    var moduleName: Boolean? = null
    var x: Int? = null
    var y: Int? = null
    var id: String? = null
}
